mlir/lib/Bindings/Python/IRAttributes.cpp

```diff
@@ -647,6 +664,55 @@ class PyDenseElementsAttribute
   static constexpr const char *pyClassName = "DenseElementsAttr";
   using PyConcreteAttribute::PyConcreteAttribute;
 
+  static PyDenseElementsAttribute
+  getFromList(py::list attributes, std::optional<PyType> explicitType,
+              DefaultingPyMlirContext contextWrapper) {
+
+    if (py::len(attributes) == 0) {
+      throw py::value_error("Attributes list must be non-empty");
+    }
```

Comment:
nit: you can drop the `{`, `}` for single line conditionals (and loops).

Reply:
Fixed